terraform/modules/k8s-karpenter/main.tf

```diff
@@ -3,8 +3,8 @@ locals {
   karpenter = {
     name          = try(var.helm.release_name, "karpenter")
     enabled       = true
-    chart         = try(var.helm.chart_name, "karpenter")
-    repository    = try(var.helm.repository, "oci://public.ecr.aws/karpenter")
+    chart         = try(var.helm.chart_name, "oci://public.ecr.aws/karpenter/karpenter")
```

Comment:
Why did you change this? What was the problem here?

Reply:
This configuration worked for github-runners that also use oci helm repository. And it's working now for deploy karpenter in Propeller project.

Reply:
I've added information about this changes in the description of the PR